components/RoomList/index.js

```diff
@@ -1,21 +1,30 @@
-import { RoomListContainer, RoomListTitle, RoomListItem } from "./styles";
+import RoomButton from "../RoomButton";
+import { RoomListContainer, RoomListItem } from "./styles";
+import Divider from "../Divider";
+import RoomIcon from "../RoomIcon";
+
 export default function RoomList() {
   // Dummy data for the rooms array
-
   const rooms = [
-    { id: 1, name: "Room 101" },
-    { id: 2, name: "Room 202" },
-    { id: 3, name: "Room 303" },
-    { id: 4, name: "Room 404" },
-    { id: 5, name: "Room 505" },
+    { id: 1, name: "Room 101", state: "Keine Angabe" },
+    { id: 2, name: "Room 202", state: "Keine Angabe" },
+    { id: 3, name: "Room 303", state: "Keine Angabe" },
+    { id: 4, name: "Room 404", state: "Keine Angabe" },
+    { id: 5, name: "Room 505", state: "Keine Angabe" },
   ];
 
   return (
     <RoomListContainer>
-      <RoomListTitle>Room List</RoomListTitle>
       <ul>
-        {rooms.map((room) => (
-          <RoomListItem key={room.id}>{room.name}</RoomListItem>
+        {rooms.map((room, index) => (
+          <div key={room.id}>
```

Comment:
The only valid child elements of a <ul> element are <li> elements. So, having a <div> directly inside a <ul> is not semantically correct. It should be: 
` <li key={room.id}>`